config/src/main/resources/org/springframework/security/config/spring-security-5.7.rnc

```diff
@@ -659,6 +659,107 @@ openid-attribute.attlist &=
 	## Specifies the number of attributes that you wish to get back. For example, return 3 emails. The default value is 1.
 	attribute count {xsd:int}?
 
+saml2-login =
+	## Configures authentication support for SAML 2.0 Login
+	element saml2-login {saml2-login.attlist}
+saml2-login.attlist &=
+	## Reference to the RelyingPartyRegistrationRepository
+	attribute relying-party-registration-repository-ref {xsd:token}?
+saml2-login.attlist &=
+	## Reference to the Saml2AuthenticationRequestFactory
+	attribute authentication-request-factory-ref {xsd:token}?
```

Comment:
I don't think we should expose `Saml2AuthenticationRequestFactory` now that it's deprecated.